egs/wsj/s5/steps/libs/nnet3/xconfig/lstm.py

```diff
@@ -846,7 +869,10 @@ def generate_lstm_config(self):
                                 abs(delay)))
         affine_str = self.config['ng-affine-options']
         lstm_str = self.config['lstm-nonlinearity-options']
-
+        lstm_dropout_value = self.config['dropout-proportion']
+        lstm_dropout_str = 'dropout-proportion='+str(self.config['dropout-proportion'])
+        lstm_dropout_per_frame_value = self.config['dropout-per-frame']
```

Comment:
I think this variable is unused, an since the name is longer than the rhs of the expression, there is not much need for it.